clang-tools-extra/test/clang-tidy/checkers/modernize/use-emplace.cpp

```diff
@@ -1285,11 +1289,9 @@ void testBracedInitTemporaries() {
 
 
   // These should not be noticed or fixed; after the correction, the code won't
-  // compile.
+  // compile in version previous to C++20.
```

Comment:
This is not needed anymore since those cases moved to another place?
```suggestion
  // compile.
```

Reply:
Indeed, forgot to delete that, thx

Reply:
All should be addressed now